langchain4j-elasticsearch/src/test/java/dev/langchain4j/store/embedding/elasticsearch/ElasticsearchEmbeddingStoreKnnWithConfigurationIT.java

```diff
@@ -0,0 +1,106 @@
+package dev.langchain4j.store.embedding.elasticsearch;
+
+import dev.langchain4j.data.embedding.Embedding;
+import dev.langchain4j.data.segment.TextSegment;
+import dev.langchain4j.model.embedding.EmbeddingModel;
+import dev.langchain4j.model.embedding.onnx.allminilml6v2q.AllMiniLmL6V2QuantizedEmbeddingModel;
+import dev.langchain4j.store.embedding.EmbeddingSearchRequest;
+import dev.langchain4j.store.embedding.EmbeddingStore;
+import org.junit.jupiter.api.*;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+
+import static dev.langchain4j.internal.Utils.randomUUID;
+import static org.assertj.core.api.Assertions.assertThat;
+
+class ElasticsearchEmbeddingStoreKnnWithConfigurationIT {
+
+    static ElasticsearchClientHelper elasticsearchClientHelper = new ElasticsearchClientHelper();
+    String indexName;
+
+    @BeforeAll
+    static void startServices() throws IOException {
+        elasticsearchClientHelper.startServices();
+        assertThat(elasticsearchClientHelper.restClient).isNotNull();
+        assertThat(elasticsearchClientHelper.client).isNotNull();
+    }
+
+    @AfterAll
+    static void stopServices() throws IOException {
+        elasticsearchClientHelper.stopServices();
+    }
+
+    @BeforeEach
+    void createEmbeddingStore() throws IOException {
+        indexName = randomUUID();
+        elasticsearchClientHelper.removeDataStore(indexName);
```

Comment:
minor: does it make sense to call remove here? `indexName` has just been generated on the line before